pkg/storage/azurecdn/storage.go

```diff
@@ -0,0 +1,87 @@
+package azurecdn
```

Comment:
can we actualy rename this to azureblob or something similiar, now we will have azurecdn for this and for meta, and none of them is really a cdn implementation

Reply:
I'd like to keep it as CDN because I actually want to get Azure CDN working in here at some point. It'll have to save to azure blob storage like it does not and then it might have to register with azure CDN